HTML/index.html

```diff
@@ -6,25 +6,32 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="description" content="Delicious' Chocolate Pizza recipe">
   <meta name="viewport" content="width=device-width, initial-scale=1">
+
+  <link href='http://fonts.googleapis.com/css?family=Yellowtail' rel='stylesheet' type='text/css'>
+  <link href='http://fonts.googleapis.com/css?family=Oswald:b' rel='stylesheet' type='text/css'>
+  <link href='http://fonts.googleapis.com/css?family=Oswald' rel='stylesheet' type='text/css'>
+  <link href='http://fonts.googleapis.com/css?family=Abel' rel='stylesheet' type='text/css'>
+  <link href='http://fonts.googleapis.com/css?family=Patrick+Hand' rel='stylesheet' type='text/css'>
+  <link href='http://fonts.googleapis.com/css?family=Droid+Serif' rel='stylesheet' type='text/css'>
+
+
   <link href="style.css" type="text/css" rel="stylesheet"/>
   <link href="960.css" type="text/css" rel="stylesheet"/>
-  <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
 </head>
 
 <body>
-  <div class="container_12 test">
-
+  <div class="container_16 test">
     <header class="masthead test">
       <section class="clearfix test">
-        <div class="grid_6 test">
+        <div class="grid_8 test">
           <img class="logo" src="file:///Users/jkoehnke/Downloads/challenge/logo.png" alt="Delicious">
-          <div class="logo-text test">
-            <h1 class="test">Delicious</h1>
-            <h6 class="test">THE BEST FOOD BLOG ON THE WEB.</h6>
+          <div class="test">
+            <h1 class="delicious test">Delicious</h1>
+            <h6 class="best test">THE BEST FOOD BLOG ON THE WEB.</h6>
```

Comment:
If you do the above, omit float: left from these two elements. The default block display of h* tags will have them be aligned on top of one another.